core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/SsccRegionEndpoint.java.tmpl

```diff
@@ -236,7 +236,11 @@ CoprocessorService, Coprocessor {
 
   private HRegionInfo regionInfo = null;
   private HRegion m_Region = null;
+#ifdef CDH5.7 APACHE1.2
+  private HRegion t_Region = null;
```

Comment:
the qualify of HRegion.getScanner(final Scan scan, final List<KeyValueScanner> scanners) has changed to public in hbase1.2, what the TransactionalRegion do before hbase1.2 is to make getScanner to be public , so in hbase1.2 it can change back to HRegion. and it may help to remove "hbase.hregion.impl" in hbase-site.xml in the future.